tests/sdk/go/go_test.go

```diff
@@ -99,6 +99,19 @@ func TestGo(t *testing.T) {
 		integration.ProgramTest(t, &options)
 	})
 
+	t.Run("Helm Skip CRD Rendering", func(t *testing.T) {
+		options := baseOptions.With(integration.ProgramTestOptions{
+			Dir:         filepath.Join("helm-skip-crd-rendering", "step1"),
+			Quick:       true,
+			SkipRefresh: true, // Istio custom resources may exhibit refresh changes.
+			ExtraRuntimeValidation: func(t *testing.T, stackInfo integration.RuntimeValidationStackInfo) {
+				assert.NotNil(t, stackInfo.Deployment)
+				assert.Equal(t, 28, len(stackInfo.Deployment.Resources))
```

Comment:
Is this right? It seems like there should only be a few resources from this chart.

Reply:
The test program actually runs identical charts with and without the CRD flag. The CRD spec I chose at random was istio which installs 20+ crds. Without the ability to disable CRDs the test would actually fail because the CRDs will end up conflicting on URNs. I made a note of it here: 
```
        // If we enabled CRDRendering on both charts, we would expect collisions on the identical
        // URNs for the installed CRDs.
        // kubernetes:apiextensions.k8s.io/v1beta1:CustomResourceDefinition virtualservices.networking.istio.io  error: Duplicate resource URN
        // 'urn:pulumi:....:helm.sh/v3:Chart$kubernetes:apiextensions.k8s.io/v1beta1:CustomResourceDefinition::virtualservices.networking.istio.io';
        // try giving it a unique name
        // See https://github.com/pulumi/pulumi-kubernetes/issues/1225
```
https://github.com/pulumi/pulumi-kubernetes/pull/1572/files#diff-9b93917e8b0cc3b2dce98335c02caff1a1f4a99e750863ddc9efda6025d851ffR36

Reply:
I reduced this down by choosing a different CRD. Due to #1225 we cant really have the same CRD being installed in the test suite - since parallel runs will conflict on URNs as well.